ci: TEST

robot-ci-heartex opened the pull request:
Hi @nikitabelonogov!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7572704408) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1669
- https://github.com/HumanSignal/label-studio-sdk/pull/175
- https://github.com/HumanSignal/dm2/pull/290

nikitabelonogov committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7572704408

nikitabelonogov committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7572704408

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7572755883

nikitabelonogov committed:
Merge branch 'develop' into 'fb-ci-test'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7572777920

nikitabelonogov committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7572777920

nikitabelonogov committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7572942107

nikitabelonogov committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7572944269

nikitabelonogov commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7708656583)
> Unresolvable merge conflict detected, please resolve it using the git command line: label_studio/frontend/dist/lsf/version.json

nikitabelonogov committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7754310055

nikitabelonogov committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7754863874

nikitabelonogov committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7754863874
